test/functional/wallet_abandonconflict.py

```diff
@@ -232,7 +232,9 @@ def run_test(self):
         balance = newbalance
 
         # Invalidate the block with the double spend. B & C's 10 BTC outputs should no longer be available
-        self.nodes[0].invalidateblock(self.nodes[0].getbestblockhash())
+        blk = self.nodes[0].getbestblockhash()
+        self.generate(self.nodes[1], 10)
```

Comment:
I've added this comment explaining why 10 blocks need to be mined:
```
# mine 10 blocks so that when the blk is invalidated, the transactions are not
# returned to the mempool
```